recipe/channels

```diff
@@ -1 +1,2 @@
 conda-forge
+maddenp
```

Comment:
I was thinking about this a bit, but it would probably be best to capture some of those options here

Reply:
Three options occur to me:
1. Copy the core `iotaa` module from its module as `iotaa.py` into this repo. This isn't great because it would complicate syncing changes and risk updates being made in `uwtools` that aren't properly upstreamed to the `iotaa` repo, leading to divergence.
2. Fork the `iotaa` repo and set it up to publish packages to the `ufs-community` anaconda.org channel. This probably makes keeping the repos in sync somewhat easier, though not completely because their CI configurations will diverge.
3. Get `iotaa` published to `conda-forge`. This would eliminate use of the `maddenp` channel, which I assume would be the source of any complaints. I don't know how much work this would take, but probably not much, assuming that whatever policies conda-forge has for inclusion could be met. This would probably mean the least amount of long-term maintenance.